lksctp-tools.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: lksctp-tools
+  version: 1.0.19
+  epoch: 0
+  description: "The Linux SCTP helper library"
+  copyright:
+    - license: GPL-2.0-only AND LGPL-2.1-only
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - libtool
+      - linux-headers
+      - m4
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/sctp/lksctp-tools
+      tag: v${{package.version}}
+      expected-commit: 05b50d379ff0037de4957bb2a1befcce88b70225
+
+  - runs: |
+      libtoolize --force --copy
+      aclocal
+      autoheader
+      automake --foreign --add-missing --copy
+      autoconf
+
+  - uses: autoconf/configure
+    with:
+      opts: |
+        --enable-shared \
+        --enable-static
+
+  - uses: autoconf/make
+    with:
+      opts: LDFLAGS=-static
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+subpackages:
+  - name: libsctp-dev
+    pipeline:
+      - uses: split/dev
+    description: libsctp dev
+
+update:
```

Comment:
They do not seem to use releases, so you probably have to use use-tags for the update section.
```
  github:
    use-tag: true
```

fwiw, you can test the update section locally like this:
```
GITHUB_TOKEN="<YOUR TOKEN HERE> wolfictl check update  --override-version 0.0.1 <yourfile>
```

Reply:
Ah nice, TIL!